4A64F3FFFE8BBC35.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "4A64F3FFFE8BBC35",
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
```

Comment:
this is the old server URL which does not work anymore, please update to V3 server